esf_torque_encoder.tex

```diff
@@ -0,0 +1,72 @@
+\documentclass{article}
+\usepackage[margin=1in, paperwidth=8.5in, paperheight=11in]{geometry}
+\usepackage{caption}
+\usepackage{subcaption}
+\usepackage{float}
+\usepackage{graphicx}
+\usepackage[utf8]{inputenc}
+\usepackage{hyperref}
+
+\begin{document}
+
+\setcounter{section}{2}
+\setcounter{subsection}{11}
+\subsection{Torque Encoder}
+\subsubsection{Description/Additional Circuitry}
+%Describe your concept of the RTDS, how is the sound produced, what are the parameters for activating the RTDS, etc.
+Two rotary potentiometers are mechanically housed in one unit from Active Sensors (P/N MHR5621) and mounted to the rotating shaft of the throttle pedal assembly. The use of a single housing eliminates concerns regarding mechanical backlash and misalignment. Each output from the potentiometers will go to a CAN connected ATmega16M1, which will compare the two outputs and send a message via CAN bus to the motor controllers with the requested torque.
+\begin{table}[H]
+	    \centering
+	        Torque encoder manufacturer and type & MHR5621 from Active Sensors  \\ \hline
+	        Torque encoder principle  & Potentiometer \\ \hline
+	         Supply Voltage & 5V\\ \hline
+	        Minimum Supply Current & 15 mV \\ \hline
+	        Operating Temperature & -55 to 150 °C \\ \hline
+	        Used Output & 0 - V \\ \hline
+	    \end{tabular}
+	    \caption{Table 27: Accelerator Pedal Position Sensor data }
+	    \label{cells}
```

Comment:
You will want to change the label here. There is already a tabel labelled "cells"